### PR TITLE
Repurpose cf-typography

### PR DESCRIPTION
**Pull request by virginiacc**
- Adds headers, links, lists, and other basic typographic patterns from cf-core and cfgov-refresh.

From `misc.less`
- `.micro-copy`
- `.short-desc`
- `.icon-link`
- `.styled-link`
- `.jump-link`
- `.block-link`

From `cf-core`
- `.pull-quote`
- `.list__branded`

From `lists.less`, everything except .filtered-by list, which seems project-specific
- `.list_link`
- `.list__unstyled`
- `.list__spaced`
- `.list_item__spaced`
- `.list__horizontal`
- `.list__icons`
- `.list__links`

From `meta.less`, everything except `.tags` (bullets have been pulled out as a pattern here & layout in cfgov-refresh)
- `.date`
- `.category-slug`
- `.header-slug`
- `.padded-header`
- `.fancy-slug`
- `.meta-header`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
   "license": "https://github.com/cfpb/cf-typography/blob/gh-pages/TERMS.md",
   "main": "src/cf-typography.less",
   "dependencies": {
-    "cf-colors": "git://github.com/cfpb/cf-colors.git",
+    "cf-core": "latest",
+    "cf-icons": "latest",
     "normalize-css": "~3.0.0",
```
Comment (himedlooff) [inline bower.json]: For consistency with the other components can you ditch the two normalize deps here please?

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-typography",
-  "version": "0.5.11",
+  "version": "0.6.0",
   "description": "Basic typography styling, including heading hierarchy, links, and some handy webfont mixins for Capital Framework.",
```
Comment (himedlooff) [inline bower.json]: This description needs updating.

File: src/cf-typography.less [hunk 3]
```diff
@@ -3,554 +3,993 @@
    Basic Typography
```
Comment (himedlooff) [inline src/cf-typography.less]: Should be updated to "Advanced Typography", or something like that.

**Comment by himedlooff** [comment]
Please update the README as well.

Everything else looks great, thanks!

**Comment by virginiacc** [comment]
Okay, I think all the fixes should be in now. Thanks!

**Comment by himedlooff** [comment]
Thanks! Tag and release and this is done.

**Comment by virginiacc** [comment]
Tagged & released. Thanks!

**Pull request merged**